web/src/pages/Home/Home.css

```
@@ -0,0 +1,160 @@
+/* ==================== Homepage Animations ==================== */
+
+@keyframes fadeInUp {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)